rustls-libssl/src/cache.rs

```diff
@@ -610,4 +633,24 @@ mod tests {
         assert!(cache.find_by_id(&[5]).is_some());
         assert!(cache.find_by_id(&[6]).is_some());
     }
+
+    #[test]
+    fn respects_context() {
+        let cache = ServerSessionStorage::new(5);
+        cache.set_context(b"hello");
```

Comment:
Should the `server.c` example perhaps also set a context in its demo session handling code?